app/models/miq_ae_yaml_export.rb

```diff
@@ -177,6 +177,7 @@ def write_all_methods(ns_fqname, class_obj)
 
   def write_method_attributes(method_obj, export_file_hash)
     envelope_hash = setup_envelope(method_obj, METHOD_OBJ_TYPE)
+    convert_playbook_attributes(envelope_hash) if method_obj.location == "playbook"
```

Comment:
@lfu @pkomanek what happens if this is the new workflow method

Reply:
This PR works only for ```playbook``` method.

The [PR](https://github.com/ManageIQ/manageiq/pull/18160) to add job/workflow template method is still pending. We can address that in a separate PR later.